src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -169,117 +293,27 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
   }
 
   const handleEditAddress = () => {
-    trackEvent({
-      action_type: "clickedValidationAddress",
-      context_module: ContextModule.ordersShipping,
-      context_page_owner_type: OwnerType.ordersShipping,
-      context_page_owner_id: contextPageOwnerSlug,
-      user_id: userId,
+    trackClickedModal({
```

Comment:
(May need to expand diff) - worth noting here that `handleBackToEdit` and `handleEditAddress` are the same button for the two different modals and have nearly identical behavior.